internal/pipe/pipe.go

```diff
@@ -25,6 +25,9 @@ var ErrSkipSignEnabled = Skip("artifact signing is disabled")
 // It means that the part of a Piper that validates some things was not run.
 var ErrSkipValidateEnabled = Skip("validation is disabled")
 
+// ErrSkipSBOMEnabled happens if --skip-sbom is set.
+var ErrSkipSBOMEnabled = Skip("artifact cataloging is disabled")
```

Comment:
not being used anywhere...

Reply:
correct! With the addition of https://github.com/goreleaser/goreleaser/pull/2648#discussion_r747133802 now it should be 👍

Reply:
seems like still unused... 

in any case, I don't think its really needed... the `Skip` method in the pipe handles it :)